cmd/dkvsrv/main.go

```diff
@@ -132,6 +132,7 @@ func main() {
 	grpcSrvr, lstnr := newGrpcServerListener()
 	defer grpcSrvr.GracefulStop()
 	srvrRole := toDKVSrvrRole(dbRole)
+	healthCheckPeriod := uint8(1)
```

Comment:
maybe move this to const ?  Also I suppose this is the ticker interval, so maybe naming it something similar might be more intuitive.

Reply:
creating a const in main and passing it onto various implementations.